spec/values/calculation/abs.hrx

```diff
@@ -0,0 +1,255 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `abs()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: abs(0)}
+
+<===> zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> positive/integer/input.scss
+a {b: abs(1)}
+
+<===> positive/integer/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> negative/decimal/input.scss
+a {b: abs(-5.6px)}
+
+<===> negative/decimal/output.css
+a {
+  b: 5.6px;
+}
+
+<===>
+================================================================================
+<===> operation/variable/input.scss
+a {
+  --test: 5;
+  b: abs(1px + 2px - var(--test))
+}
+
+<===> operation/variable/output.css
+a {
+  --test: 5;
+  b: abs(3px - var(--test));
+}
+
+<===>
+================================================================================
+<===> operation/preserves_units/input.scss
+a {b: abs(-7px / 4em) * 1em}
+
+<===> operation/preserves_units/output.css
+a {
+  b: 1.75px;
+}
+
+<===>
+================================================================================
+<===> operation/incompatible_units/input.scss
+a {b: abs(17px * 5%)}
+
+<===> operation/incompatible_units/error
+Error: 85px*% isn't a valid CSS value.
+  ,
+1 | a {b: abs(17px * 5%)}
+  |       ^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
```

Comment:
I don't think we need to test this separately from the above. The previous test already demonstrates that `abs()` passes through units as-is; this error is just Sass complaining about the units we already know `abs()` returns.